scout/adapter/mongo/user.py

```diff
@@ -87,8 +87,12 @@ def user(self, email=None, user_id=None):
             query['_id'] = user_id
         else:
             LOG.info("Fetching user %s", email)
-            query['email'] = email
-            
+            query = {
+                '$or' : [
```

Comment:
As it stands, this will not change anything, but the idea is right.
We could as for *$and: _id:email, email:email* and fall back to email:email if that doesn't help. I'll update!